share/spice/statista/content.handlebars

```diff
@@ -0,0 +1 @@
+<pre>{{name}}</pre>
```

Comment:
This isn't necessary, the template does not allow you to specify a `content` sub-template :)